### PR TITLE
Euen.26.rebar3

### PR DESCRIPTION
**Pull request by Euen**

### CODE REVIEW

File: src/kafkerl_error.erl [hunk 1]
```diff
@@ -43,6 +44,7 @@ get_error_name(?NOT_COORDINATOR_FOR_CONSUMER_CODE) ->
 get_error_name(?UNKNOWN) ->
   "Unknown".
 
+-spec get_error_description(integer()) -> string().
```
Comment (gadgetci) [inline src/kafkerl_error.erl]: According to **Dialyzer**:
> Type specification kafkerl_error:get_error_description(integer()) -> string() is a supertype of the success typing: kafkerl_error:get_error_description(integer()) -> [1..255,...]

File: src/kafkerl_error.erl [hunk 2]
```diff
@@ -8,6 +8,7 @@
 %%==============================================================================
 %% API
 %%==============================================================================
+-spec get_error_name(integer()) -> string().
```
Comment (gadgetci) [inline src/kafkerl_error.erl]: According to **Dialyzer**:
> Type specification kafkerl_error:get_error_name(integer()) -> string() is a supertype of the success typing: kafkerl_error:get_error_name(integer()) -> [1..255,...]

File: src/kafkerl.erl [hunk 3]
```diff
@@ -38,18 +38,24 @@
 %%==============================================================================
 %% API
 %%==============================================================================
+-spec start() -> ok | {error, term()}.
```
Comment (gadgetci) [inline src/kafkerl.erl]: According to **Dialyzer**:
> The specification for kafkerl:start/0 states that the function might also return {'error',_} but the inferred return is 'ok'

File: elvis.config [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    {config,
+     [#{dirs => ["src", "test"],
+        filter => "*.erl",
+        rules => [ {elvis_style, invalid_dynamic_call, #{ignore => [kafkerl_utils]}}
+                 , {elvis_style, nesting_level, #{level => 4}}
+                 , {elvis_style, dont_repeat_yourself, #{min_complexity => 15}}
+                 ],
+        ruleset => erl_files
+       },
+      #{dirs => ["."],
```
Comment (elbrujohalcon) [inline elvis.config]: We don't have makefiles, do we?

**Pull request merged**